src/Evolution/Systems/CurvedScalarWave/Worldtube/ElementActions/SendToWorldtube.hpp

```diff
@@ -87,62 +85,44 @@ struct SendToWorldtube {
       Parallel::GlobalCache<Metavariables>& cache,
       const ArrayIndex& /*array_index*/, const ActionList /*meta*/,
       const ParallelComponent* const /*meta*/) {
-    const auto& puncture_field =
-        db::get<Worldtube::Tags::PunctureField<Dim>>(box);
-    if (puncture_field.has_value()) {
-      const auto& element_id = db::get<domain::Tags::Element<Dim>>(box).id();
-      const auto& excision_sphere = db::get<Tags::ExcisionSphere<Dim>>(box);
-      const auto direction = excision_sphere.abutting_direction(element_id);
-      ASSERT(direction.has_value(), "Should be abutting");
+    const auto& element_id = db::get<domain::Tags::Element<Dim>>(box).id();
+    const auto& excision_sphere = db::get<Tags::ExcisionSphere<Dim>>(box);
+    const auto direction = excision_sphere.abutting_direction(element_id);
+    if (direction.has_value()) {
```

Comment:
[optional] You can avoid the indent by just returning early if the direction doesn't have a value